db/index.js

```diff
@@ -0,0 +1,9 @@
+const Sequelize = require('sequelize');
+const DataTypes = require('sequelize');
+
```

Comment:
I believe there's a way to use destructuring for lines 1 & 2 to abide by the DRY principle. Here's the link for clarification, it's right under "Tip for reading the docs" - https://sequelize.org/master/manual/getting-started.html

Looks like we're able to do --> const { Sequelize, DataTypes } = require('sequelize');

Let me know what you think

Reply:
Good call. I think for this, I'm not actually using the `DataTypes const` at all , so I'll just drop line 2 here, & in bundles.model.js, I'll swap to destructuring.